docs/proposal-rose-suite-run.md

```diff
@@ -24,27 +22,75 @@ The logical step is therefore to move these extra functionalities provided by
 Rose into Cylc - and retire `rose suite-run` and related utilities - so that
 all suites will start up with only Cylc commands in the future.
 
+## Work Plan
+
+- [ ] Agree future form of `cylc-flow.rc` [Cylc-Admin #43](https://github.com/cylc/cylc-admin/issues/43)
+  - [x] Create new issue against cylc-admin
+  - [ ] Create a template `flow.rc`, listing all the options
+    to be available in the combined `flow.rc`. [Link to it](example_files/flow.rc)
+  - [ ] Deprecate settings for job host?
+  - [ ] Add sections for `[job platforms]`
+
+
+- [ ] Implement new `cylc-flow.rc` form. [Cylc-flow #3260](https://github.com/cylc/cylc-flow/issues/3260)
+  - [ ] Check old tests for `global.rc` & `suite.rc` to ensure that functionality
+    is not lost.
+  - [ ] Devise tests for the new flow.rc
+  - [ ] Create new config parser, perhaps called `cylc-flow-rc-parser.py`
+
+
+- [ ] Implement Cluster support functionality. [Cylc-flow #2199](https://github.com/cylc/cylc-flow/issues/2199)
+  - [ ] Modify `task_job_mgr.py` to use the new variables.
+  - [ ] Randomize login host used
+
+
+- [ ] Agree names and functionality of future `cylc flow`(working name)
+  command and command args.[#1030](https://github.com/cylc/cylc-flow/issues/1030)
+  - [ ] Summarize debate on ticket.
+  - [ ] Create a PR against cylc-admin creating a specification for `cylc flow`
+
+
+- [ ] Create a `cylc flow` CLI.
+  - [ ] Include suite validation in `cylc flow` CLI.
+
+
+- [ ] Replace `rose-suite.conf`.
```

Comment:
I feel like the bottom half of this list is increasingly sketchy. I'd be happier of people with bigger domain knowledge might care to colour it in a bit...